test/functorch/test_eager_transforms.py

```diff
@@ -3089,8 +3089,10 @@ def test_resnet18_per_sample_grads(self, device):
         func_model, weights = make_functional(model)
 
         def compute_loss(weights, image, target):
-            output = func_model(weights, images)
```

Comment:
this was previously using the captured, batched value images instead of the BatchedTensor image